### ISSUE TITLE
"Eye in Speech Bubble" detection as a sequence 1f441-fe0f-200d-1f5e8-fe0f is broken

### ISSUE THREAD

**Issue by seiya-git** [Ranks/emojione#453]
**Expected result:**
```
<img class="emojione" alt="👁🗨" src="//cdn.jsdelivr.net/emojione/assets/png/1f441-1f5e8.png?v=2.2.7"/>
```
**Actual result:**
```
<img class="emojione" alt="👁" src="//cdn.jsdelivr.net/emojione/assets/png/1f441.png?v=2.2.7"/>️‍🗨️
```
**emojione version:** 2.2.7

**Sources:**
http://www.unicode.org/Public/emoji/2.0/emoji-zwj-sequences.txt
http://www.unicode.org/Public/emoji/4.0/emoji-zwj-sequences.txt

**Comment by caseyahenson** [comment]
@seiya-git Good catch! This will be corrected in our upcoming 3.0 release.

**Comment by caseyahenson** [comment]
@seiya-git this has been fixed with our 3.0 release.
